conftest.py

```diff
@@ -4,12 +4,18 @@
 
 import pytest
 from django.contrib.auth import get_user_model
+from django.test import Client
 
 from core.models import Community, Post
 
 User = get_user_model()
 
 
+@pytest.fixture()
+def client() -> type[Client]:
```

Comment:
Why? 
we have this: https://pytest-django.readthedocs.io/en/latest/helpers.html#client-django-test-client

Reply:
I mean, pytest-django, the package we have already, is providing this. We don't need second one in this fixture